LiskHQ/lisk-hub
# Migrate dashboard to the new design - closes #1936

### What issue have I solved?

-- #1936 

### How have I implemented/fixed it?

Update the routes file for replace the actual dashboard to the new dashboard.
Bookmark will display based on the active token.
Wallet details will show ONLY when user signed in.
Recent transactions (last 5) will display based on active token.

Recent transaction logic update to display information properly when active token change

**NOTE:**
./src/bookmarks folder needs to be deleter after implement Bookmarks page

### How has this been tested?


1. Login into the app.
2. Check the component wallet details show properly information same as in the top bar for LSK and BTC.
3. Check the list of recent tx, tops 5 tx will be shown.
4. Community feed show properly the information and click on each message will open a new window.
5. Bookmarks component shows a max of 5 bookmarks.
6. Click on any bookmark will redirect to the specific account.

The dashboard show information for LSK and BTC so for see the BTC part you need to Add btc to localStorage: **localStorage.setItem('btc', true);** and for switch between BTC and LSK you need to dispatch this function from the redux devTool in the browser.

For LSK
`{type:'SETTINGS_UPDATED', data: { token: { active: 'LSK' } } }`

For BTC
`{type:'SETTINGS_UPDATED', data: { token: { active: 'BTC' } } }`

**NOTE:** For now the View All button in bookmarks component will he hide because there is no bookmark page for now, once we implement bookmarks page the button will be enable it and redirect to that page


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
